INSTALL.md

```diff
@@ -32,6 +32,16 @@ Please refer to instructions provided by the repository.
 
 - lablgtk3 and its prerequisites (ocamlfind, GTK 3 and its dependencies)
 
+##### Optional, for building the user manual
+
+- LaTeX
+- dvips
+- ps2pdf (included with Ghostscript, for example)
+- HEVEA (https://hevea.inria.fr/) (optional, for HTML and text formats)
+- Lynx (optional, for text format)
+- (for developers only) HEVEA and Lynx are required to re-build the manual
```

Comment:
I think this means "regen strings" needs these.   I feel like I'm nitpicking but it would be nice to have these hoisted to a section that addresses what it takes to prepare a PR.   Maybe that's just something earlier that explains that the manual dependencies are not optional for PR submission (but I guess latex/dvips/ps2pdf are not actually needed for that).

Reply:
Yes, it's the "regen strings" thing.
Agree, but I think such information is better kept in HACKING.md or CONTRIBUTING.md or similar. If this line is too provocative for this document then I can just remove it.

> but I guess latex/dvips/ps2pdf are not actually needed for that

I think so, but not sure. dvips/ps2pdf should definitely not be needed. I don't think hevea needs latex, but haven't checked.